ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_path_selection.schema.yml

```diff
@@ -124,11 +124,50 @@ keys:
             name:
               type: str
               description: Load-balance policy name.
+            hop_count_lower:
+              type: bool
+              description: Prefer paths with lowest hop-count.
+            jitter:
+              type: int
+              description: Jitter requirement for this load balance policy in milliseconds.
+              convert_types:
+                - str
+              min: 0
+              max: 10000
+            latency:
+              type: int
+              description: One way delay requirement for this load balance policy in milliseconds.
+              convert_types:
+                - str
+              min: 0
+              max: 10000
+            loss_rate:
+              type: str
+              description: Loss Rate requirement for this load balance policy in milliseconds.
+              convert_types:
+                - int
+                - float
+              pattern: "^\\d+(\\.\\d{1,2})?$"
             path_groups:
               type: list
+              primary_key: name
               description: List of path-groups to use for this load balance policy.
               items:
-                type: str
+                type: dict
+                keys:
+                  name:
+                    type: str
+                    description: Path-group name
+                  priority:
+                    type: int
+                    description: |-
+                      Priority for this path-group.
+                      Default is 1 and not rendred in EOS cli.
```

Comment:
Normally we render whatever the user gives, and do not have a default for eos_cli_config_gen. Then we write `The EOS default value is 1`